spec/parser_spec.rb

```diff
@@ -16,4 +18,20 @@
       expect(first_line).to eq(FIRST_LINE)          
     }.not_to raise_error
   end
+
```

Comment:
@feperessim  por convenção, nós criamos um describe com o nome da função no seguinte formato:

```
describe '#read_first_line' do
....
end
```

E colocamos cada it correspondente ao seu describe. No momento é muito dificil visualizar o que você está testando.

`#` significa que a função é pública
`.` significa que a função é estática

https://www.betterspecs.org/#describe

Reply:
Obrigado pela indicação. Correção realizada.